e2e/playwright/playwright.config.ts

```diff
@@ -12,7 +12,7 @@ export default defineConfig({
     globalSetup: require.resolve('./global_setup'),
     forbidOnly: testConfig.isCI,
     outputDir: './test-results',
-    testDir: 'tests',
+    testDir: process.env.PLAYWRIGHT_TEST_DIR || 'tests',
```

Comment:
Please move to [test.config.ts](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/playwright/test.config.ts#L51-L54) as common entry point from env variables. Then here could be: `testDir: testConfig.testDir,`.
For env name, please change to `PW_TEST_DIR`.